docs/user/security/audit-logging.asciidoc

```diff
@@ -34,3 +34,130 @@ multiple audit events. The following is a list of the events that can be generat
 | `saved_objects_authorization_failure`    | Logged when a user isn't authorized to access a saved
                                              objects when using a role with <<kibana-privileges>>
 |======
+
```

Comment:
Earlier in this file, there is a reference to writing to the standard output. We should probably update that to indicate that it's only true for the legacy audit logger:

> Audit logging uses the standard Kibana logging output, which can be configured in the kibana.yml and is discussed in Configure Kibana.